.github/workflows/production.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy to AWS ECS(Production)
+
+on:
+  push:
+    tags:
+      - 'v*'  # Trigger on any new tag that starts with 'v' (e.g., v1.0.0, v2.0.1)
```

Comment:
According to Semtech the `v` is not required. I used to love the `v` before but is easy to forget. 
This comment is open to discussion.

Reply:
I make this little change and merge